pandaserd/module.py

```diff
@@ -196,7 +195,7 @@ def create_rel(self, left_table_name, right_table_name, left_on=None, right_on=N
             self.table_gen_code.append(rel)
 
 
-    def write_to_file(self, filename='output.txt'):
+    def write_to_file(self, filename='output.gv'):
```

Comment:
To be honest I cannot recall why I made this change specifically. Probably because [`.gv`](https://www.reviversoft.com/en/file-extensions/gv) is a standard extension for Grapvhiz DOT files. Likely I was using another tool that would pick up `.gv` files and render them.

Reply:
Yep, `.gv` is the correct Graphviz DOT files. Graphviz is imported, but not sure if it's on the build yet...

Reply:
If the output is saved as a .gv file, graphviz library can be used to display it on the screen if using a notebook and can also be used to save it, as a png in this example.

```python
...
raw_entity_relationship_diagram.write_to_file('raw_entity_relationship_diagram.gv')
graph = graphviz.Source.from_file('raw_entity_relationship_diagram.gv')
graph.render('erd_raw', format='png') #Save file
display(graph)
```